parts/linux/cloud-init/artifacts/ci-syslog-watcher.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o pipefail
+
+[ ! -f "/var/run/mdsd/update.status" ] && exit 0
```

Comment:
I think, plan was to use container  insights specific path  to avoid any conflicts with MDSD, right? are you planning to update this path to container insights specific path for example, /var/run/containerinsights/update.status?

Reply:
yes that is the plan. this is a draft PR to test things out. I'll have the actual paths in next iteration